x/mongo/driver/connstring/connstring.go

```diff
@@ -143,12 +145,26 @@ func (u *ConnString) Validate() error {
 // to the server.
 type ConnectMode uint8
 
+var _ fmt.Stringer = ConnectMode(0)
+
 // ConnectMode constants.
 const (
 	AutoConnect ConnectMode = iota
 	SingleConnect
 )
 
+// String implements the fmt.Stringer interface.
+func (c ConnectMode) String() string {
```

Comment:
This was added to make the error message readable if a user specifies conflicting values for `connect` and `directConnection`.